### PR TITLE
Update replication controller pages to v1beta3 in web ui

### PR DESCRIPTION
**Pull request by bcbroussard**
Depends on #8304 and #8332

cc @jackgr @lavalamp @bgrant0607 @brendandburns

### CODE REVIEW

File: www/app/assets/js/app.js [hunk 1]
```diff
@@ -205,7 +206,7 @@ app.controller('TabCtrl', [
       .service('cAdvisorService', ["$http", "$q", "ENV", function($http, $q, ENV) {
         var _baseUrl = function(minionIp) {
           var minionPort = ENV['/']['cAdvisorPort'] || "8081";
-          var proxy = ENV['/']['cAdvisorProxy'] || "/api/v1beta2/proxy/nodes/";
+          var proxy = ENV['/']['cAdvisorProxy'] || "/api/v1beta3/proxy/nodes/";
```
Comment (caesarxuchao) [inline www/app/assets/js/app.js]: I suspect it should be "/api/v1beta3/proxy/namespaces/default/nodes/"

**Comment by bcbroussard** [comment]
@caesarxuchao any way you can merge in #8332 since it is ready to go and has status/lgtm?

**Comment by caesarxuchao** [comment]
LGTM

**Comment by caesarxuchao** [comment]
@bcbroussard, I think only @dchen1107 can merge now.

**Comment by bcbroussard** [comment]
@caesarxuchao I bet she's very popular right now. Is the merge freeze still partially in effect then?

**Comment by bgrant0607** [comment]
e2e is broken. Merges are on hold.

**Comment by bcbroussard** [comment]
thanks @bgrant0607, I saw the email yesterday but wasn't sure if it was still going on

**Comment by thockin** [comment]
rebase please

**Comment by bgrant0607** [comment]
I'd like to get all of the v1beta3-related PRs ASAP, prior to the 0.18 cut. Please rebase.

**Comment by bcbroussard** [comment]
@thockin @bgrant0607 rebased. 

I'll need to rebase after each one, so after this is merged, please ping me on the PR you want to merge next.

**Comment by thockin** [comment]
you could pre-rebase the rest of these if you felt clever...

**Comment by bcbroussard** [comment]
@thockin rebased and ready after #8611 goes in

**Comment by bgrant0607** [comment]
Integration test failure looks like a flake. Restarted Travis.

**Pull request merged**